Casks/graalvm-jdk17.rb

```diff
@@ -0,0 +1,49 @@
+cask "graalvm-jdk17" do
+  arch arm: "aarch64", intel: "x64"
+
+  version "17.0.7"
+  sha256 arm:   "cb45f6585ef02134a6a6ffb6de20db96197486ffef8821ad97b11fe2fc0c23b8",
+         intel: "905255762546c69e3bb8d815a5d20e2e3cfa5332b868ab90af7aa0afe21e74ea"
+
+  installation_dir = "graalvm-jdk-#{version}+8.1".freeze
```

Comment:
Where does `8.1` come from?

Reply:
This is the build number (JDK dot GraalVM).

Reply:
JDK == 8?

Reply:
8 is the JDK build number (17.0.7+**8**), 1 is the GraalVM build number.